envi/codeflow.py

```diff
@@ -154,10 +155,10 @@ def addCodeFlow(self, va, arch=envi.ARCH_DEFAULT):
             try:
                 op = self._mem.parseOpcode(va, arch=arch)
             except envi.InvalidInstruction, e:
-                print 'parseOpcode error at 0x%.8x: %s' % (va,e)
+                logging.getLogger("envi/codeflow.addCodeFlow").warning('parseOpcode error at 0x%.8x: %s', (va,e))
                 continue 
             except Exception, e:
-                print 'parseOpcode error at 0x%.8x: %s' % (va,e)
+                logging.getLogger("envi/codeflow.addCodeFlow").warning('parseOpcode error at 0x%.8x: %s', (va,e))
```

Comment:
Ah, good catch. I'll patch those up. 

-----Original Message-----
From: "atlas0fd00m" <notifications@github.com>
Sent: ‎10/‎30/‎2015 8:03
To: "vivisect/vivisect" <vivisect@noreply.github.com>
Cc: "Willi Ballenthin" <willi.ballenthin@gmail.com>
Subject: Re: [vivisect] Replace some calls to `print()` with appropiate callsto loggers (#85)

In envi/codeflow.py:
>                  continue 
>              except Exception, e:
> -                print 'parseOpcode error at 0x%.8x: %s' % (va,e)
> +                logging.getLogger("envi/codeflow.addCodeFlow").warning('parseOpcode error at 0x%.8x: %s', (va,e))
this as well: no tuple.
—
Reply to this email directly or view it on GitHub.